Tests/TuistGeneratorTests/Generator/ProjectGeneratorTests.swift

```diff
@@ -51,10 +51,11 @@ final class ProjectGeneratorTests: XCTestCase {
     func test_generate_scheme() throws {
```

Comment:
minor: It's nice to have each test focus on one aspect to make it clear what's being tested and in the event it fails it

e.g. `test_generate_sharedSchemes()` and `test_generate_localScheme()`

Reply:
Agree with this one

Reply:
This is what I love to do